cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -1,12 +1,51 @@
-const checkout = new AdyenCheckout(window.Configuration);
+var checkoutConfiguration = window.Configuration;
+var isValid = false;
+var storeDetails;
+
+checkoutConfiguration.paymentMethodsConfiguration =
+    {
+        card: {
+            enableStoreDetails: showStoreDetails,
+            onBrand: function (brandObject) {
+                $('#cardType').val(brandObject.brand);
+            },
+            onFieldValid: function (data) {
+                if(data.endDigits){
+                    maskedCardNumber = MASKED_CC_PREFIX + data.endDigits;
+                    $("#cardNumber").val(maskedCardNumber);
+                }
+            },
+            onChange: function (state) {
+                storeDetails = state.data.storePaymentMethod;
+                isValid = state.isValid;
+                var type = state.data.paymentMethod.type;
+                $('#browserInfo').val(JSON.stringify(state.data.browserInfo));
+                componentArr[type].isValid = isValid;
+                componentArr[type].stateData = state.data;
+            }
+        },
+        boletobancario: {
+            personalDetailsRequired: true, // turn personalDetails section on/off
+            billingAddressRequired: false, // turn billingAddress section on/off
+            showEmailAddress: false, // allow shopper to specify their email address
+
+            // Optionally prefill some fields, here all fields are filled:
+            data: {
+                socialSecurityNumber: '56861752509',
+                firstName: 'José',
+                lastName: 'Silva',
```

Comment:
can we prefill this data with the actual firstname lastname filled into the checkout ?